packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -78,8 +78,7 @@ export class PostPublishPanel extends Component {
 				<div className="editor-post-publish-panel__header">
 					{ isPostPublish ? (
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td>
<img width="278" alt="post publish before" src="https://github.com/user-attachments/assets/f582e90a-be23-473f-b732-58a0776d1952">


</td>
    <td>

<img width="276" alt="post publish panel after" src="https://github.com/user-attachments/assets/6688b364-46fc-42ed-ba9b-fa1830191374">

</td>
  </tr>
</table>

Reply:
The screenshot may be misleading, the button in question is actually the close button (focused in the following screenshot):

![Screenshot 2024-09-06 at 12 44 27](https://github.com/user-attachments/assets/158deb49-229e-4337-9de2-68bb0dbf4f24)

My suggestion here is to add `size="compact"`

```suggestion
							size="compact"
```

Reply:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td>

<img width="281" alt="Screenshot 2024-09-12 at 1 36 33 AM" src="https://github.com/user-attachments/assets/bafaa1ef-ab2f-4e87-9ef4-1245570f7e3c">


</td>
    <td>

<img width="282" alt="Screenshot 2024-09-12 at 1 31 57 AM" src="https://github.com/user-attachments/assets/b5df533d-7240-4626-868e-933f4bd2d37f">
</td>
  </tr>
</table>


Hii @ciampo @mirka 
Added size="compact" for close button. 
PFA : New screenshots for before and after. 
Before is with __next40pxDefaultSize prop
After is with size="compact"